auth/routes/oauth_routes.go

```diff
@@ -6,10 +6,12 @@ const (
 	OAuthBegin    = "OAuthBegin"
 	OAuthCallback = "OAuthCallback"
 	LogOut        = "LogOut"
+	PostCustom    = "PostCustom"
 )
 
 var OAuthRoutes = rata.Routes{
 	{Path: "/auth/logout", Method: "GET", Name: LogOut},
 	{Path: "/auth/:provider/callback", Method: "GET", Name: OAuthCallback},
 	{Path: "/auth/:provider", Method: "GET", Name: OAuthBegin},
+	{Path: "/auth/:provider/", Method: "POST", Name: PostCustom},
```

Comment:
The routes are evaluated here: https://github.com/concourse/atc/pull/169/files#diff-a0a797688867b8d2b038ac457091c196R84
If a provider provides no routes (default `nil`) then the endpoint will not be available